bigtable/spark/src/main/scala/example/Wordcount.scala

```diff
@@ -0,0 +1,55 @@
+package example
+
+import com.google.cloud.bigtable.hbase.BigtableConfiguration
+import org.apache.hadoop.hbase.HConstants
+import org.apache.hadoop.hbase.client.Put
+import org.apache.hadoop.hbase.io.ImmutableBytesWritable
+import org.apache.hadoop.hbase.mapreduce.TableOutputFormat
+import org.apache.hadoop.hbase.util.Bytes
+import org.apache.spark.SparkContext
+
+object Wordcount extends App {
+
+  val projectId = args(0)
+  val instanceId = args(1)
+  val table = args(2)
+  val file = args(3)
+
+  val ColumnFamily = "cf"
+  val ColumnFamilyBytes = Bytes.toBytes(ColumnFamily)
+  val ColumnNameBytes = Bytes.toBytes("Count")
+
+  import org.apache.spark.SparkConf
+  val sparkConf = new SparkConf()
+
+  // Workaround for a bug in TableOutputFormat in HBase 1.6.0
```

Comment:
I thought we are using hbase 2.x?

Reply:
think it's a leftover from the days I used 1.x. Left FIXME to verify it.